keyboards/keebio/iris/keymaps/shinze/keymap.c

```diff
@@ -0,0 +1,173 @@
+#include QMK_KEYBOARD_H
+
+#include "keymap_french_osx.h"
+#include "keymap_bepo.h"
+#include "sendstring_bepo.h"
+
+extern keymap_config_t keymap_config;
+
+#define _BEPO    0
```

Comment:
Non-consecutive layer numbers will generate extra empty layers in between, unnecessarily increasing flash usage. The better way to define layers is with an enum, eliminating the need to number them manually:

```c
enum layer_names {
    _BEPO,
    _LOWER,
    _RAISE,
    _ADJUST
};
```